pkg/media/ivf-writer_test.go

```diff
@@ -0,0 +1,151 @@
+package media
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/pions/rtp"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestIVFWriter_FileCreation(t *testing.T) {
+	assert := assert.New(t)
+
+	// Create a IVFWriter with an empty filename
+	writer, err := NewIVFWriter("")
+	assert.NotNil(err, "NewIVFWriter shouldn't create file with an empty name")
+	assert.Nil(writer, "NewIVFWriter should return a nil value for the writer, in case of error")
+
+	// Creates a valid file, and tries to open it twice
+	writer, err = NewIVFWriter("/tmp/pions_testCreateIVFWriter1.ivf")
```

Comment:
Is there any better way to test that wouldn't imply the creation of a file ? (via mocking the `open()` method, as you suggested ?)